views/search_results.erb

```diff
@@ -10,11 +11,11 @@
       </tr>
       <% flights.each do |flight| %>
         <tr>
-          <td><%= Time.parse(flight.takeoff_at).strftime('%I:%M %p') %></td>
-          <td><%= Time.parse(flight.land_at).strftime('%I:%M %p') %></td>
+          <td><%= TimeMath.datetime_for_airport(date, flight.takeoff_at, flight.orig).strftime('%I:%M %p %Z') %></td>
+          <td><%= TimeMath.datetime_for_airport(date, flight.land_at, flight.dest).strftime('%I:%M %p %Z') %></td>
```

Comment:
This logic isn't too crazy for a template but a good challenge here would be to think about what kind of messages you are sending and what you are trying to state.

```
TimeMath.datetime_for_airport(date, flight.land_at, flight.dest).strftime('%I:%M %p %Z')
```

It feels like you might want to say something like

```
class FlightPresenter
  def initialize(flight)
    @flight = flight
  end

  attr_reader :flight

  def arrival_time_at_destination
    TimeMath.datetime_for_airport(date, flight.land_at, flight.dest).strftime('%I:%M %p %Z')
  end
end
```

Changing your template code to:

```
<% flights.each do |flight| %>
   <% flight_info = FlightPresenter.new(flight) %>
  <!-- other ERB -->
  <td><%= flight_info.arrival_time_at_destination %></td>
```

You may even want to move the flight to flight presenter into your controller (AKA `get '/search'`)